common/reviews/api/ts-command-line.api.md

```diff
@@ -142,10 +153,8 @@ export abstract class CommandLineParser extends CommandLineParameterProvider {
     protected _getArgumentParser(): argparse.ArgumentParser;
     protected onExecute(): Promise<void>;
     selectedAction: CommandLineAction | undefined;
-    readonly toolDescription: string;
-    readonly toolFilename: string;
```

Comment:
@sachinjoseph  This is a breaking API change. Why did you remove them?

Reply:
I removed it because they're never being assigned or used.

Reply:
@octogonz Shall I resolve this?